hartshorn-core/src/main/java/org/dockbox/hartshorn/core/domain/Exceptional.java

```diff
@@ -51,6 +52,7 @@
 
     private static final Exceptional<?> EMPTY = new Exceptional<>();
 
+    @Nullable
```

Comment:
If the parameter for `throwable` is `@Nullable` at line 484, also make sure the field is marked as nullable, as well as getters for both fields (`#error`, `#type`). Also ensure `#get` is annotated with `@NotNull`

Reply:
`#error` throws a `NoSuchElementException` if it doesn't exist, so shouldn't it be annotated with `@NotNull` too?

Reply:
Indeed you're right, I see you've added it correctly in your last commit, thanks!